tests/playwright/pages/elementor-panel-tabs/content.js

```diff
@@ -41,11 +41,13 @@ export default class Content {
 		await this.editorPage.getPreviewFrame().locator( EditorSelectors.siteTitle ).click();
 	}
 
-	async verifyLink( element, options = { target, href, rel, customAttributes } ) {
+	async verifyLink( element, options = { target, href, rel, customAttributes, widget } ) {
```

Comment:
Please add a JSDoc comment specifying the parameters and their types, this is getting confusing

Reply:
`<a href="http://www.link.test" target="_blank" rel="nofollow" mykey="testValue">Add Your Heading Text Here</a>`
`options = { target, href, rel, customAttributes, widget }`
If you look at `a` tag, this is just a standard HTML attribute of this tag. A widget is just a widget name. I don't know how to name it more understandable. 

I have added jsdoc, but this is an extra 10 lines of comments.